app/controllers/placements/schools/placements_controller.rb

```diff
@@ -6,8 +6,13 @@ class Placements::Schools::PlacementsController < Placements::ApplicationControl
   helper_method :edit_attribute_path, :add_provider_path, :add_mentor_path
 
   def index
-    @pagy, placements = pagy(placements_scope.includes(:subject, :mentors, :additional_subjects, :provider).order("subjects.name"))
-    @placements = placements.decorate
+    @pagy, scoped_placements = pagy(
+      placements
+        .where(academic_year: academic_year_scope)
+        .includes(:subject, :mentors, :additional_subjects, :provider)
+        .order("subjects.name"),
+    )
+    @placements = scoped_placements.decorate
```

Comment:
It's a shame there's no `@placements.decorate!` method which would avoid needing to re-assign the variable.

But I wonder if this might be cleaner to avoid the awkwardness of needing a name for the intermediate variable? I think it should work.

```suggestion
    @pagy, @placements = pagy(
      placements
        .where(academic_year: academic_year_scope)
        .includes(:subject, :mentors, :additional_subjects, :provider)
        .order("subjects.name"),
    )
    @placements = @placements.decorate
```